### DISCUSSION TITLE
csv output from API

### DISCUSSION THREAD

**Discussion by ghughes888** [open-meteo/open-meteo#313] [Q&A]
Is it possible to obtain a csv file rather than a json file when calling the API? I want to call the API from Stata which does not handle json files well. The website offers the option of obtaining either xlsx or csv files as output, so I had assumed that there was a way of changing the default output format. However, I can't find a parameter in the API specification that would allow one to control the form of the output.

**Comment by patrick-zippenfenig** [comment]
Hi, if you append `&format=csv` the API returns data in CSV
